style.css

```diff
@@ -858,6 +873,112 @@ body {
 
 .Arrow2 {
   position: absolute;
-  top: 3831px;
+  margin-top: 27px;
   margin-left: 281px;
 }
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-image: url(image/final_contact_background.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 375px;
+  height: 750px;
+  margin-top: 50px;
+  border-top-left-radius: 50px;
+  background-color: #6070ff;
+}
+
+.contact-form h2 {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #fff;
+  margin-top: 110px;
+  padding: 10px;
+}
+
+.contact-form p {
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  color: #ebebff;
+  text-align: center;
+  margin-top: 16px;
+  height: 120px;
+  width: 327px;
+
+  /* margin-left: -25px; */
+  padding: 20px;
+}
```

Comment:
- [ ] Kindly use the same font-family in the Figma template.
- [ ] Your `h2` and `p` tags are being overlapped by the background image and the form, and its making hard to read the text. Kindly fix that.
![image](https://user-images.githubusercontent.com/61919617/172641843-43b57227-c8d2-42a6-82a4-8d728c724d32.png)
![image](https://user-images.githubusercontent.com/61919617/172642015-738236fe-5c89-42f2-bbe2-4041b231f951.png)